prompting/llms/vllm_llm.py

```diff
@@ -44,15 +44,17 @@ def load_vllm_pipeline(model_id: str, device: str, mock=False):
         return MockPipeline(model_id)
 
     # Calculates the gpu memory utilization required to run the model within 20GB of GPU
-    max_allowed_memory_in_gb = 20
+    max_allowed_memory_in_gb = 60
     max_allowed_memory_allocation_in_bytes = max_allowed_memory_in_gb * 1e9
     gpu_mem_utilization = calculate_gpu_requirements(
         device, max_allowed_memory_allocation_in_bytes
     )
 
     try:
         # Attempt to initialize the LLM
-        return LLM(model=model_id, gpu_memory_utilization=gpu_mem_utilization)
+        _ = LLM(model=model_id, gpu_memory_utilization = gpu_mem_utilization, quantization="AWQ")
+        _.llm_engine.tokenizer.eos_token_id = 128009
```

Comment:
Got from @bkb2135 that this is an work around solution to prevent loading the tokenizer every time we want to put in the custom eos token, and apparently this is required for llama3 generation.